tests/test_storageclass_invalid.py

```diff
@@ -46,8 +46,19 @@ def test_storageclass_cephfs_invalid(invalid_cephfs_storageclass, tmpdir):
     logger.debug(f"Status of PVC {pvc_name} after creation: {pvc_status}")
     assert pvc_status == 'Pending'
 
-    logger.info('Wait for 60 seconds')
-    time.sleep(60)
+    try:
+        logger.info('Wait for 60 seconds')
+        pvc_status_changed = pvc.wait_for_resource_status(
```

Comment:
now if you'll git fetch you can use a much better version of this function 'wait_for_resource()'

Reply:
ok, updated